Zinc/Private/ZincRepo.m

```diff
@@ -34,8 +34,10 @@
 #define DOWNLOADS_DIR @"zinc/downloads"
 #define REPO_INDEX_FILE @"repo.json"
 
-static NSMutableDictionary* _ReposByURL;
+static const float kContentBundleFlushLimitInMegabytes = 100.f;
+static NSString * const kContentBundlePrefix = @"com.wonder.content";
```

Comment:
this is actually griffin specific, so it should exist there and be passed in somehow...